src/app/containers/CpsAssetPageMain/index.jsx

```diff
@@ -50,7 +50,7 @@ const CpsAssetPageMain = ({ pageData }) => {
         description={summary}
         openGraphType="website"
       />
-      <LinkedData type="Article" seoTitle={title} />
+      <LinkedData type="Article" seoTitle={title} headline={title} />
```

Comment:
Just wondering, the issue says the headline should be derived from _headline OR seoHeadline_, but I can only see this using the former, is that deliberate?

Reply:
So... it turns out that AReS returns different fields, depending on whether the article was created in Optimo or CPS.  If you look at https://github.com/bbc/simorgh/blob/d1e372efa71c608370c91276752bb6c602bf1357/src/app/containers/CpsAssetPageMain/index.jsx#L21, the title is derived from the promo.headlines.headline

For an Optimo article, the SEO headline is promo.headlines.seoHeadline

For me, this is not ideal, and I would have preferred if AReS returned seoHeadline regardless of the source of the assert. This solution is a bit of a workaround.

Should I ask AReS whether it would be possible for them to add seoHeadline to the schema for CPS assets too?  My only concern is that it might take longer to get this done, so we would have to put this on hold until the work had been completed.

Reply:
> For an Optimo article, the SEO headline is promo.headlines.seoHeadline
Ah sorry, I just looked at the table in the issue and assumed that we were being expected to use either of those two fields on CPS pages depending on which was available. I didn't realise it was covering the differences on article pages as well. Makes sense now :-)